src/site/content/en/blog/load-faster-like-proxx/index.md

```diff
@@ -127,13 +110,7 @@ Looking at the waterfall, we can see that once the first JavaScript file is done
 Let's take a look at what our changes have achieved. It's important to not change any other variables in our test setup that could skew the results, so we will be using [WebPageTest's simple setup][wpt simple] for the rest of this article and look at the filmstrip:
 
 <figure class="w-figure">
-  <video controls muted
-    preload="metadata"
-    class="w-screenshot"
-    poster="https://storage.googleapis.com/web-dev-assets/js-heavy-sites/preload-proxx-load-poster.jpg"
-      src="https://storage.googleapis.com/web-dev-assets/js-heavy-sites/preload-proxx-load.mp4"
-      type="video/mp4; codecs=h264">
-  </video>
+  {% Video src="video/tcFciHGuF3MxnTr1y5ue01OGLBn2/v76UWV9zidMILuFlLpaX.mp4", src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/zjJ410ZPSr99njy4KMoh.jpg", controls=true, muted=true, preload="metadata", class="w-screenshot" %}
```

Comment:
Should these have two src arguments? It seems like the second src argument is for the poster image. It looks like there are a few instances of this.